app/components/components.md

```diff
@@ -0,0 +1,141 @@
+**ALL PAGES**
+
+* NAV component:
+- site name
+- cart icon
+- login button || username
+
+* FOOTER component?
+
+**PRODUCTS PAGE**
+
+* FILTER component:
+- Categories
+- Price ranges
+- keyword
+
+* ALL PRODUCTS component:
+- pagination
+- sort by
+- image
+- title
+- price
+- '+' (add to cart)
+- more info
+
+**INDIVIDUAL PRODUCT PAGE**
+
+* PRODUCT component:
+- image
+- title
+- price
+- description
+- add to cart (disabled if product is sold out)
```

Comment:
Should we have a quantity input here too?

Reply:
sure